testem.browserstack.js

```diff
@@ -58,7 +58,7 @@ const BrowserStackLaunchers = {
 };
 
 module.exports = {
-  test_page: 'dist/tests/index.html?hidepassed&hideskipped&timeout=60000',
+  test_page: 'dist/tests/index.html?hidepassed&hideskipped&timeout=60000&disable_local_debug',
```

Comment:
All these testem changes are using a new flag form glimmer-vm which revealed a real (existing) bug that we need to fix.

Some info here: https://github.com/emberjs/ember.js/issues/20573

Reply:
Is the "local debug" code still shipped with ember-source or do they get stripped out somewhere?

Reply:
It's stripped out with `DEBUG` from `@glimmer/env`

Reply:
_HOWEVER_, DEBUG is passed on to consumers of ember-source, so I think glimmer-vm needs to compile this away

Reply:
done!

Hopefully we can return the opt-in capability once we add import.meta.env everywhere, hopefully